src/llm_vm/server/anarchparse.py

```diff
@@ -0,0 +1,23 @@
+import argparse, configparser
+
+
+parser = argparse.ArgumentParser()
+
+# Set parse strategy here, lines should be self explanatory
+parser.add_argument("-b", "--big_model", type=str, default='chat_gpt', help='Big LLM Model. Default: chat_gpt')
+parser.add_argument("-c", "--config_file", type=str, help='Config File')
+parser.add_argument("-p", "--port", type=int, default=3002, help='Port Number. Default: 3002')
+parser.add_argument("-s", "--small_model", type=str, default='gpt', help='Small LLM Model. Default: gpt')
+parser.add_argument("-H", "--host", type=str, default='127.0.0.1', help='Host Address. Default: 127.0.0.1')
+
```

Comment:
i think we should have some way to list registered models? for ease of use , we can do that later